### PR TITLE
build: change files related to the creation of the windows installer

### PR DESCRIPTION
**Pull request by emanuelaepure10**
The following changes have been done to solve the windows installer build error:
- HALE.product: replace '»' with the encoding character
- build-msi.xml: add encoding UTF-8 to all the replace sequences into file
- release.md add more details in how to create the windows installer 
and changelog.md text has been align with the release 5.0.1 text.

ING-3831

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 ### Added
 - Add an option to overwrite content in Geopackage Writer instead of appending content
 - Add a parameter to the GeoPackage writer that allows creating tables for all mapping-relevant target types
-- Add the creation of a .cpg file when a .shp file is beeing exported
+- Create a code page (.cpg) file when exporting a Shapefile
```
Comment (florianesser) [inline CHANGELOG.md:12]: This change is unrelated to the fixes for the Windows Installer and shouldn't be in the same commit.

File: RELEASE.md [hunk 2]
```diff
@@ -56,6 +56,9 @@ On master branch:
 
 After the release
 =================
-1. Create Windows installer
+1. Create Windows installer.
+Prerequisites: install Wix Toolset v.3.11 availabe here https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm
```
Comment (florianesser) [inline RELEASE.md:60]: Instead of pasting raw URLs, please use MD links, i.e. `[available here](https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm)`.

File: RELEASE.md [hunk 3]
```diff
@@ -56,6 +56,9 @@ On master branch:
...
+And from https://github.com/halestudio/hale/tree/master/build run:
+`build.bat product -o windows -a x86_64 HALE`
```
Comment (florianesser) [inline RELEASE.md:62]: `build.bat` is available in the root directory, so maybe:

```
Then run `build.bat product -o windows -a x86_64 HALE` from the root directory.
```

File: CHANGELOG.md (hunk 1 above)
Comment (emanuelaepure10) [reply CHANGELOG.md:12]: @florianesser I know...do I need to create a pull request just for a file? Or maybe this should go to the release branch?

Comment (florianesser) [reply CHANGELOG.md]: > do I need to create a pull request just for a file? 

Not necessarily, but it should at least be in a separate commit.

> Or maybe this should go to the release branch?

The change should go into `master`. As the 5.0.1 release is already tagged and built, I would be in favour of not re-opening it for this editorial change.

Comment (florianesser) [reply CHANGELOG.md:12]: @emanuelaepure10 The commit message `refactor! feat: Release hale»studio 5.0.1` is not consistent with the developer guidelines. For a fixup commit you would use `fixup! feat: Release hale»studio 5.0.1`.

**Review by florianesser — APPROVED** [review]
LGTM

**Review by florianesser — CHANGES_REQUESTED** [review]
@emanuelaepure10 Please make sure that there are not merge commits in the commit list before merging/rebasing this into master

**Comment by emanuelaepure10** [comment]
@florianesser I just create a new PR from a new branch. I will close this PR.
Thank you for your help
